### PR TITLE
feat: detect the lowest Kubernetes version in upgrade-k8s CLI command

### PR DESCRIPTION
**Pull request by Unix4ever**
Scan all pods in `kube-system` and find `kube-proxy`, `kube-scheduler`,
`kube-controller-manager` and `kube-apiserver` ones, then check the
lowest version amogst them.

Signed-off-by: Artem Chernyshev <artem.0xD2@gmail.com>

### CODE REVIEW

**Review by AlekSi — APPROVED** [review]
LGTM with a small style suggestion: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

File: pkg/cluster/kubernetes/detect.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	}
+
+	version, err := semver.NewVersion(options.ToVersion)
+	if err != nil {
+		return "", err
+	}
+
+	for _, pod := range pods.Items {
+		app := pod.GetObjectMeta().GetLabels()["k8s-app"]
+		if _, ok := apps[app]; ok {
```
[the remark this answers is not in the document]
Comment (Unix4ever) [reply pkg/cluster/kubernetes/detect.go:43]: :facepalm:

File: pkg/cluster/kubernetes/detect.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			for _, container := range pod.Spec.Containers {
+				if container.Name == app {
```
[the remark this answers is not in the document]
Comment (Unix4ever) [reply pkg/cluster/kubernetes/detect.go:45]: :facepalm: thanks!

**Pull request merged**